maxim/tests/test_connection_retry_demo.py

```
@@ -0,0 +1,253 @@
+from unittest.mock import Mock, patch
+from urllib3.exceptions import ProtocolError
+from requests.exceptions import ConnectionError
+from http.client import RemoteDisconnected
+
+# Add the maxim package to the path
+import sys
+import os
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "."))
+
+from maxim.apis.maxim_apis import MaximAPI
```

Pylint (3.3.7)

[error] Unable to import 'maxim.apis.maxim_apis'

(E0401)